roles/openshift_provisioners/tasks/main.yaml

```diff
@@ -12,6 +12,19 @@
   check_mode: no
   tags: provisioners_init
 
+- name: Set default image variables based on openshift_deployment_type
+  include_vars: "{{ var_file_name }}"
+  with_first_found:
+    - "{{ openshift_deployment_type }}.yml"
+    - "default_images.yml"
+  loop_control:
+    loop_var: var_file_name
+
+- name: Set eventrouter image facts
+  set_fact:
```

Comment:
I prefer not to use set_fact unless it's 100% necessary.  This means that some value we dynamically derive is need across play scopes.

I prefer to use dictionaries for this type of small thing:

https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_facts/defaults/main.yml#L4-L15